wrappers-docker/Makefile

```diff
@@ -1,5 +1,5 @@
 IMAGE_NAME=docker.io/seldonio/core-python-wrapper
-IMAGE_VERSION=0.7
+IMAGE_VERSION=0.8
```

Comment:
Have we updated all docs to version 0.8?

Reply:
Not yet. These changes only impact someone who wants to build and wrap an outlier detector, but this isn't document anywhere at the moment...